backend/push.js

```diff
@@ -199,11 +199,19 @@ export const postEvent = async (subscription: Object, event: ?string): Promise<v
 
   if (!req.ok) {
     const endpointHost = new URL(subscription.endpoint).host
-    console.warn(`Error ${req.status} sending push notification to '${subscription.id}' via ${endpointHost}`)
+    console.warn(
+      await req.text().then(text => {
```

Comment:
This is missing the second argument to `then` or a call to `catch`.

Reply:
what exactly would the `.catch` be for? `.text()` failing?

Reply:
For example. If it fails, the app will currently crash.

Reply:
I'd replace the whole thing with `await req.text().catch((e) => '(error parsing body)')`